<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LT-CHN-33875" name="restassured.automation.testcases.Restassured_Automation_ProcedureGroups" tests="12" failures="1" timestamp="13 Jun 2021 05:30:12 GMT" time="590.747" errors="0">
  <testcase name="ProcedureGroup_PostCreateAProcedureGroup_status400" time="44.049" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroup_PutCreateAProcedureGroup_status400" time="38.873" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_DeleteCreateAProcedureGroupTree_status400" time="47.103" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroup_PutCreateAProcedureGroup_status200" time="63.673" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroup_GetCreateAProcedureGroup_status200" time="45.068" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_PutCreateAProcedureGroupTree_status204" time="50.288" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_DeleteCreateAProcedureGroupTree_status204" time="50.618" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroup_PostCreateAProcedureGroup_status200" time="43.603" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_PostCreateAProcedureGroupTree_status204" time="47.874" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_PutCreateAProcedureGroupTree_status400" time="74.523" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroupTree_PostCreateAProcedureGroupTree_status400" time="43.727" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="ProcedureGroup_PostCreateAProcedureGroup_status409" time="41.348" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups">
    <failure type="java.lang.AssertionError" message="expected [409] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at restassured.automation.testcases.Restassured_Automation_ProcedureGroups.ProcedureGroup_PostCreateAProcedureGroup_status409(Restassured_Automation_ProcedureGroups.java:1262)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ProcedureGroup_PostCreateAProcedureGroup_status409 -->
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_ProcedureGroups -->
